index.html

```diff
@@ -1,11 +1,215 @@
 <!DOCTYPE html>
 <html lang="en">
-    <head>
-        <meta charset="utf-8">
-        <title>Saritasa recommended packages for PHP</title>
-    </head>
-    <body>
-        <h1>Saritasa recommends</h1>
-        <div>Content will be here...</div>
-    </body>
+<head>
+  <meta charset="utf-8">
+  <title>Saritasa recommended packages for PHP</title>
+  <link
+      rel="stylesheet"
+      href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css"
+      integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u"
+      crossorigin="anonymous">
+
+  <script
+      src="https://code.jquery.com/jquery-3.3.1.min.js"
+      integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8="
+      crossorigin="anonymous">
+  </script>
+
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"
```

Comment:
Use material design components (ex. https://vuetifyjs.com/en/ ), not bootstrap - it was explicitly stated in task.